Frontier machine enviroment optimizations

sarats opened the pull request:
Just opening a PR to see what integrating this PR would entail.

Other than machine files, the Scorpio submodule and changes to 
`components/elm/src/biogeophys/BandDiagonalMod.F90` are the other major bits.

Should switch to cli115 instead of cli115-ecp as needed. May also need Scorpio update for ADIOS output.

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - User Requested Retest - Label AT: RETEST will be reset after testing.

E3SM-Autotester commented:
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection is Not Necessary for this Pull Request.

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Failure: Timed out waiting for job SCREAM_PullRequest_Autotester_Mappy to start: Total Wait = 1803
- Other jobs have been previously started - We must stop them...

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - User Requested Retest - Label AT: RETEST will be reset after testing.

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Mappy</h3>
<ul>
  <li>Build Num: 4332</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;Frontier</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2503</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>43b56072652957dcf13c07d3e648d92206478346</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>2aa2fe86d7d43edb3d2e9a6ac5c055bbd63088e7</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 4861</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;Frontier</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2503</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>43b56072652957dcf13c07d3e648d92206478346</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>2aa2fe86d7d43edb3d2e9a6ac5c055bbd63088e7</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: SCREAM (E3SM-Project/scream)</dt>
  <ul>
    <li>Branch: sarats/frontier-gb</li>
    <li>SHA: 43b56072652957dcf13c07d3e648d92206478346</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: sarats
</details>

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Jenkins Testing: 1 or more Jobs FAILED

Note: Testing will normally be attempted again in approx. 2 Hrs. If a change to the PR source branch occurs, the testing will be attempted again on next available autotester run.

<details>
<summary>
  <b>Pull Request Auto Testing has FAILED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Mappy</h3>
<ul>
  <li>Build Num: 4332</li>
  <li>Status: FAILED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;Frontier</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2503</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>43b56072652957dcf13c07d3e648d92206478346</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>2aa2fe86d7d43edb3d2e9a6ac5c055bbd63088e7</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 4861</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;Frontier</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2503</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>43b56072652957dcf13c07d3e648d92206478346</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>2aa2fe86d7d43edb3d2e9a6ac5c055bbd63088e7</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>
</details>


<details>
<summary>
  <b>Console Output (last 100 lines) : SCREAM_PullRequest_Autotester_Mappy # 4332</b> (click to expand)
</summary>
<pre>
<code>
b'Starting MODEL_BUILD for test ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 with 16 procs'
b'Finished SHAREDLIB_BUILD for test PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 55.834119 seconds (PASS)'
b'Starting MODEL_BUILD for test PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 with 16 procs'
b'Finished SHAREDLIB_BUILD for test ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 in 108.903541 seconds (PASS)'
b'Finished SHAREDLIB_BUILD for test ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 125.882135 seconds (PASS)'
b'Starting MODEL_BUILD for test ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 with 16 procs'
b'Finished MODEL_BUILD for test ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 in 142.618412 seconds (PASS)'
b'Starting RUN for test ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 in 0.880944 seconds (PEND). [COMPLETED 1 of 6]'
b'Starting MODEL_BUILD for test ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 with 16 procs'
b'Finished MODEL_BUILD for test SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 in 143.673498 seconds (PASS)'
b'Starting RUN for test SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 in 0.814762 seconds (PEND). [COMPLETED 2 of 6]'
b'Finished MODEL_BUILD for test ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 in 202.332959 seconds (PASS)'
b'Starting RUN for test ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 in 6.999551 seconds (PEND). [COMPLETED 3 of 6]'
b'Finished MODEL_BUILD for test PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 211.710683 seconds (PASS)'
b'Starting RUN for test PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 2.466700 seconds (PEND). [COMPLETED 4 of 6]'
b'Finished MODEL_BUILD for test ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 in 404.734400 seconds (PASS)'
b'Starting RUN for test ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 in 0.816101 seconds (PEND). [COMPLETED 5 of 6]'
b'Finished MODEL_BUILD for test ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 464.382252 seconds (PASS)'
b'Starting RUN for test ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 with 1 proc on interactive node and 64 procs on compute nodes'
b'Finished RUN for test ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 in 0.840363 seconds (PEND). [COMPLETED 6 of 6]'
b'Waiting for tests to finish'
b'DIFF ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2.C.20230901_010522_9fpxvb'
b'DIFF ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9.C.20230901_010522_9fpxvb'
b"test-scheduler took 821.9660997390747 seconds'"
b"+ errors='Waiting for tests to finish"
b'DIFF ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2.C.20230901_010522_9fpxvb'
b'DIFF ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9.C.20230901_010522_9fpxvb'
b"test-scheduler took 821.9660997390747 seconds'"
b"+ V1_FAILURES_DETAILS+='Waiting for tests to finish"
b'DIFF ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2.C.20230901_010522_9fpxvb'
b'DIFF ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9.C.20230901_010522_9fpxvb'
b"test-scheduler took 821.9660997390747 seconds'"
b'+ set +x'
b'######################################################'
b'FAILS DETECTED:'
b'  SCREAM V1 TESTING FAILED!'
b'Waiting for tests to finish'
b'DIFF ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_D_Lh4.ne4_ne4.F2010-SCREAMv1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_D_Ln21.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.scream-rad_frequency_2.C.20230901_010522_9fpxvb'
b'DIFF ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/PET_Ln9_P32x2.ne4pg2_ne4pg2.F2010-SCREAMv1.mappy_gnu9.C.20230901_010522_9fpxvb'
b'DIFF SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9 (phase BASELINE)'
b'    Case dir: /ascldap/users/e3sm-jenkins/acme/scratch/SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.mappy_gnu9.C.20230901_010522_9fpxvb'
b'test-scheduler took 821.9660997390747 seconds'
b'######################################################'
b"Build step 'Execute shell' marked build as failure"
b'Performing Post build task...'
b'Match found for : : True'
b'Logical operation result is TRUE'
b'Running script  : #!/bin/bash -le\r'
b'\r'
b'cd $WORKSPACE/${BUILD_ID}/\r'
b'\r'
b'./scream/components/eamxx/scripts/jenkins/jenkins_cleanup.sh'
b'[SCREAM_PullRequest_Autotester_Mappy] $ /bin/bash -le /tmp/jenkins13390921011436981821.sh'
b'POST BUILD TASK : FAILURE'
b'END OF POST BUILD TASK : 0'
b'Sending e-mails to: lbertag@sandia.gov'
b'Finished: FAILURE'
b''
</code>
</pre>
</details>

<details>
<summary>
  <b>Console Output (last 100 lines) : SCREAM_PullRequest_Autotester_Weaver # 4861</b> (click to expand)
</summary>
<pre>
<code>
b'        Start  91: cosp_standalone_np1'
b' 91/105 Test  #91: cosp_standalone_np1 .....................................   Passed    2.27 sec'
b'        Start  92: shoc_cld_p3_rrtmgp_np1'
b' 92/105 Test  #92: shoc_cld_p3_rrtmgp_np1 ..................................   Passed    3.98 sec'
b'        Start  93: shoc_cld_spa_p3_rrtmgp_np1'
b' 93/105 Test  #93: shoc_cld_spa_p3_rrtmgp_np1 ..............................   Passed   35.93 sec'
b'        Start  94: shoc_p3_subcycled'
b' 94/105 Test  #94: shoc_p3_subcycled .......................................   Passed    2.77 sec'
b'        Start  95: shoc_p3_monolithic'
b' 95/105 Test  #95: shoc_p3_monolithic ......................................   Passed    1.66 sec'
b'        Start  96: check_subcycling'
b' 96/105 Test  #96: check_subcycling ........................................   Passed    0.05 sec'
b'        Start  97: homme_shoc_cld_p3_rrtmgp_np1'
b' 97/105 Test  #97: homme_shoc_cld_p3_rrtmgp_np1 ............................   Passed    7.80 sec'
b'        Start  98: model_baseline'
b' 98/105 Test  #98: model_baseline ..........................................   Passed    8.20 sec'
b'        Start  99: model_initial'
b' 99/105 Test  #99: model_initial ...........................................   Passed    4.96 sec'
b'        Start 100: model_restart'
b'100/105 Test #100: model_restart ...........................................   Passed    5.69 sec'
b'        Start 101: restarted_vs_monolithic_check_np1'
b'101/105 Test #101: restarted_vs_monolithic_check_np1 .......................   Passed    0.09 sec'
b'        Start 102: homme_shoc_cld_spa_p3_rrtmgp_np1'
b'102/105 Test #102: homme_shoc_cld_spa_p3_rrtmgp_np1 ........................   Passed   36.95 sec'
b'        Start 103: homme_shoc_cld_spa_p3_rrtmgp_128levels_np1'
b'103/105 Test #103: homme_shoc_cld_spa_p3_rrtmgp_128levels_np1 ..............   Passed   39.32 sec'
b'        Start 104: homme_shoc_cld_spa_p3_rrtmgp_128levels_tend_check_np1'
b'104/105 Test #104: homme_shoc_cld_spa_p3_rrtmgp_128levels_tend_check_np1 ...   Passed    1.12 sec'
b'        Start 105: homme_shoc_cld_spa_p3_rrtmgp_pg2_np1'
b'105/105 Test #105: homme_shoc_cld_spa_p3_rrtmgp_pg2_np1 ....................   Passed   24.37 sec'
b''
b'100% tests passed, 0 tests failed out of 100'
b''
b'Label Time Summary:'
b'bfbhash             =   0.97 sec*proc (1 test)'
b'check               =   1.63 sec*proc (1 test)'
b'cld                 = 149.47 sec*proc (7 tests)'
b'cld_fraction        =   1.99 sec*proc (1 test)'
b'cosp                =   2.27 sec*proc (1 test)'
b'diagnostics         =  36.76 sec*proc (20 tests)'
b'driver              = 251.41 sec*proc (21 tests)'
b'dynamics            = 121.74 sec*proc (8 tests)'
b'fail                =  52.39 sec*proc (4 tests)'
b'horiz_remap         =   1.84 sec*proc (1 test)'
b'io                  =  33.38 sec*proc (7 tests)'
b'io,remap            =   3.74 sec*proc (1 test)'
b'p3                  = 357.18 sec*proc (15 tests)'
b'physics             = 600.84 sec*proc (28 tests)'
b'physics_nudging     =   1.79 sec*proc (1 test)'
b'rrtmgp              = 157.38 sec*proc (13 tests)'
b'shoc                = 307.73 sec*proc (12 tests)'
b'spa                 = 112.29 sec*proc (5 tests)'
b'surface_coupling    =   2.53 sec*proc (1 test)'
b'tms                 =  24.37 sec*proc (1 test)'
b''
b'Total Test time (real) = 994.93 sec'
b''
b'The following tests did not run:'
b'\t 27 - physics_saturation_baseline_fake (Disabled)'
b'\t 33 - p3_baseline_f90_fake (Disabled)'
b'\t 34 - p3_baseline_cxx_fake (Disabled)'
b'\t 41 - shoc_baseline_f90_fake (Disabled)'
b'\t 42 - shoc_baseline_cxx_fake (Disabled)'
b'==============================================================================='
b"Testing '\\''983f5e2cd2312bb4ac20d831a70a661fa83c5c27'\\'' for test '\\''full_sp_debug'\\''"
b'==============================================================================='
b'RUN: taskset -c 52-103 sh -c \'\\\'\'SCREAM_BUILD_PARALLEL_LEVEL=52 CTEST_PARALLEL_LEVEL=1 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx/ctest-build/full_sp_debug/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx/ctest-build/full_sp_debug -DBUILD_NAME_MOD=full_sp_debug -S /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx/cmake/ctest_script.cmake -DCTEST_SITE=weaver -DCMAKE_COMMAND="-C /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx/cmake/machine-files/weaver.cmake -DNetCDF_Fortran_PATH=/projects/ppc64le-pwr9-rhel8/tpls/netcdf-fortran/4.6.0/gcc/11.3.0/openmpi/4.1.4/5ka6asw -DNetCDF_C_PATH=/projects/ppc64le-pwr9-rhel8/tpls/netcdf-c/4.9.0/gcc/11.3.0/openmpi/4.1.4/mdd6fth -DCMAKE_BUILD_TYPE=Debug -DEKAT_DEFAULT_BFB=True -DSCREAM_DOUBLE_PRECISION=False -DEKAT_DISABLE_TPL_WARNINGS=\'\\\'\'\\\'\\\'\'\'\\\'\'ON\'\\\'\'\\\'\\\'\'\'\\\'\' -DCMAKE_CXX_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpicxx -DCMAKE_C_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpicc -DCMAKE_Fortran_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=1 -DSCREAM_TEST_DATA_DIR=/home/projects/e3sm/scream/pr-autotester/master-baselines/weaver/full_sp_debug/data" \'\\\'\''
b'FROM: /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx/ctest-build/full_sp_debug'
b'==============================================================================='
b"Testing '\\''983f5e2cd2312bb4ac20d831a70a661fa83c5c27'\\'' for test '\\''release'\\''"
b'==============================================================================='
b'RUN: taskset -c 104-155 sh -c \'\\\'\'SCREAM_BUILD_PARALLEL_LEVEL=52 CTEST_PARALLEL_LEVEL=1 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx/ctest-build/release/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx/ctest-build/release -DBUILD_NAME_MOD=release -S /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx/cmake/ctest_script.cmake -DCTEST_SITE=weaver -DCMAKE_COMMAND="-C /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx/cmake/machine-files/weaver.cmake -DNetCDF_Fortran_PATH=/projects/ppc64le-pwr9-rhel8/tpls/netcdf-fortran/4.6.0/gcc/11.3.0/openmpi/4.1.4/5ka6asw -DNetCDF_C_PATH=/projects/ppc64le-pwr9-rhel8/tpls/netcdf-c/4.9.0/gcc/11.3.0/openmpi/4.1.4/mdd6fth -DCMAKE_BUILD_TYPE=Release -DEKAT_DISABLE_TPL_WARNINGS=\'\\\'\'\\\'\\\'\'\'\\\'\'ON\'\\\'\'\\\'\\\'\'\'\\\'\' -DCMAKE_CXX_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpicxx -DCMAKE_C_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpicc -DCMAKE_Fortran_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=1 -DSCREAM_TEST_DATA_DIR=/home/projects/e3sm/scream/pr-autotester/master-baselines/weaver/release/data" \'\\\'\''
b'FROM: /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx/ctest-build/release'
b'==============================================================================='
b"Testing '\\''983f5e2cd2312bb4ac20d831a70a661fa83c5c27'\\'' for test '\\''full_debug'\\''"
b'==============================================================================='
b'RUN: taskset -c 0-51 sh -c \'\\\'\'SCREAM_BUILD_PARALLEL_LEVEL=52 CTEST_PARALLEL_LEVEL=1 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx/ctest-build/full_debug/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx/ctest-build/full_debug -DBUILD_NAME_MOD=full_debug -S /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx/cmake/ctest_script.cmake -DCTEST_SITE=weaver -DCMAKE_COMMAND="-C /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx/cmake/machine-files/weaver.cmake -DNetCDF_Fortran_PATH=/projects/ppc64le-pwr9-rhel8/tpls/netcdf-fortran/4.6.0/gcc/11.3.0/openmpi/4.1.4/5ka6asw -DNetCDF_C_PATH=/projects/ppc64le-pwr9-rhel8/tpls/netcdf-c/4.9.0/gcc/11.3.0/openmpi/4.1.4/mdd6fth -DCMAKE_BUILD_TYPE=Debug -DEKAT_DEFAULT_BFB=True -DEKAT_DISABLE_TPL_WARNINGS=\'\\\'\'\\\'\\\'\'\'\\\'\'ON\'\\\'\'\\\'\\\'\'\'\\\'\' -DCMAKE_CXX_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpicxx -DCMAKE_C_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpicc -DCMAKE_Fortran_COMPILER=/projects/ppc64le-pwr9-rhel8/tpls/openmpi/4.1.4/gcc/11.3.0/base/vu2aei6/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=1 -DSCREAM_TEST_DATA_DIR=/home/projects/e3sm/scream/pr-autotester/master-baselines/weaver/full_debug/data" \'\\\'\''
b'FROM: /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx/ctest-build/full_debug'
b'OVERALL STATUS: PASS'
b'Starting analysis on weaver with cmd: cd /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx && source /etc/profile.d/modules.sh && module purge && module load cmake/3.25.1 git/2.39.1 python/3.10.8 py-netcdf4/1.5.8 gcc/11.3.0 cuda/11.8.0 openmpi netcdf-c netcdf-fortran parallel-netcdf netlib-lapack && true && bsub -I -q rhel8 -n 4 -gpu num=4 ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -p -i -m weaver'
b'RUN: cd /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx && source /etc/profile.d/modules.sh && module purge && module load cmake/3.25.1 git/2.39.1 python/3.10.8 py-netcdf4/1.5.8 gcc/11.3.0 cuda/11.8.0 openmpi netcdf-c netcdf-fortran parallel-netcdf netlib-lapack && true && bsub -I -q rhel8 -n 4 -gpu num=4 ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -p -i -m weaver'
b'FROM: /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/4861/scream/components/eamxx'
b"Completed analysis on weaver'"
b'+ [[ 0 != 0 ]]'
b'+ [[ 1 == 0 ]]'
b'+ [[ weaver == \\m\\a\\p\\p\\y ]]'
b'+ set +x'
b'Performing Post build task...'
b'Match found for : : True'
b'Logical operation result is TRUE'
b'Running script  : #!/bin/bash -le\r'
b'\r'
b'cd $WORKSPACE/${BUILD_ID}/\r'
b'\r'
b'./scream/components/eamxx/scripts/jenkins/jenkins_cleanup.sh'
b'[SCREAM_PullRequest_Autotester_Weaver] $ /bin/bash -le /tmp/jenkins9518255554805193799.sh'
b'POST BUILD TASK : SUCCESS'
b'END OF POST BUILD TASK : 0'
b'Sending e-mails to: lbertag@sandia.gov'
b'Finished: SUCCESS'
b''
</code>
</pre>
</details>

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Failure: Timed out waiting for job SCREAM_PullRequest_Autotester_Mappy to start: Total Wait = 1803
- Other jobs have been previously started - We must stop them...

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Mappy</h3>
<ul>
  <li>Build Num: 4333</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>Frontier</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2503</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>43b56072652957dcf13c07d3e648d92206478346</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>2aa2fe86d7d43edb3d2e9a6ac5c055bbd63088e7</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 4862</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>Frontier</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2503</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>43b56072652957dcf13c07d3e648d92206478346</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>2aa2fe86d7d43edb3d2e9a6ac5c055bbd63088e7</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: SCREAM (E3SM-Project/scream)</dt>
  <ul>
    <li>Branch: sarats/frontier-gb</li>
    <li>SHA: 43b56072652957dcf13c07d3e648d92206478346</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: sarats
</details>

sarats commented:
@AaronDonahue Did you get a chance to run a CESS test using these machine settings?

ndkeen commented on components/elm/src/biogeophys/BandDiagonalMod.F90:
```diff
@@ -218,4 +218,1891 @@ subroutine BandDiagonal(bounds, lbj, ubj, jtop, jbot, numf, filter, nband, b, r,
+!! reference impl
+!! from https://netlib.org/cgi-bin/netlibfiles.txt?format=txt&filename=/lapack/lapack_routine/dgbsv.f
+!! license https://netlib.org/lapack/LICENSE.txt
+
```
This is what I thought would be better

sarats commented:
Reverted Land BandDiagonal changes and Scorpio module update. 
Just FYI, rebase is causing many conflicts, so reverted these specific commits using `git revert --strategy resolve commit`. 

Left https://github.com/E3SM-Project/scream/pull/2503/commits/4dbb07fe1c31ed0ea20c7511bacde6be80a33234 as it just reduces opt-level for the file. 
```
[sarat@login13 scream ]$ git revert --strategy resolve 4dbb07fe1c31ed0ea20c7511bacde6be80a33234
CONFLICT (modify/delete): cime_config/machines/Depends.frontier-scream-gpu.crayclang-scream.cmake deleted in parent of 4dbb07fe1c (custom flags for banddiag file) and modified in HEAD. Version HEAD of cime_config/machines/Depends.frontier-scream-gpu.crayclang-scream.cmake left in tree.
```

sarats commented:
This includes a couple of commits applied as a patch that were since then merged into master in https://github.com/E3SM-Project/scream/pull/2431

sarats commented:
Btw, I don't know about the provenance of this 'inout' issue in ELM.
https://github.com/E3SM-Project/scream/pull/2503/commits/95729a37a1cdd987c375129a13bc4c8a5326a941

Is this needed or should be reverted?

AaronDonahue commented:
> @AaronDonahue Did you get a chance to run a CESS test using these machine settings?

@sarats , I have not.  Although I think either @ndkeen or @crterai would be better suited to give this branch a try on `frontier`.

AaronDonahue commented:
>but could @AaronDonahue double check if they are the expected baseline diffs?

@ambrad , I don't see any reason in this PR that we should see DIFF's on `mappy`.

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - User Requested Retest - Label AT: RETEST will be reset after testing.

E3SM-Autotester commented:
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection is Not Necessary for this Pull Request.

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Mappy</h3>
<ul>
  <li>Build Num: 4370</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;Frontier</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2503</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>2eb7a4a65a44295b196e11bd9d30dd4a26d89072</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>2aa2fe86d7d43edb3d2e9a6ac5c055bbd63088e7</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 4897</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;Frontier</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2503</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>2eb7a4a65a44295b196e11bd9d30dd4a26d89072</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>2aa2fe86d7d43edb3d2e9a6ac5c055bbd63088e7</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: SCREAM (E3SM-Project/scream)</dt>
  <ul>
    <li>Branch: sarats/frontier-gb</li>
    <li>SHA: 2eb7a4a65a44295b196e11bd9d30dd4a26d89072</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: sarats
</details>

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Jenkins Testing: all Jobs PASSED

<details>
<summary>
  <b>Pull Request Auto Testing has PASSED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Mappy</h3>
<ul>
  <li>Build Num: 4370</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;Frontier</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2503</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>2eb7a4a65a44295b196e11bd9d30dd4a26d89072</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>2aa2fe86d7d43edb3d2e9a6ac5c055bbd63088e7</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 4897</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST;Frontier</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2503</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>2eb7a4a65a44295b196e11bd9d30dd4a26d89072</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>2aa2fe86d7d43edb3d2e9a6ac5c055bbd63088e7</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>
</details>

E3SM-Autotester commented:
All Jobs Finished; status = PASSED, However Inspection must be performed before merge can occur...

crterai commented:
I ran a short ne1024 simulation where I merged the Frontier machine environment on top of the machines/frontier branch and it ran into the following crash pasted below 1.5 days (1479 steps) in to simulation. I've resubmitted to see if it's reproducible. 
The branch used is: https://github.com/E3SM-Project/scream/commits/machines/frontier_w_machenv_opt
And the git hash: `c407a49af9` 
Run directory: `/lustre/orion/cli115/proj-shared/terai/e3sm_scratch/cess-v2test_baselineForPerf.ne1024pg2_ne1024pg2.F2010-SCREAMv1.20230909/run/`
Run directory for baseline: `/lustre/orion/cli115/proj-shared/terai/e3sm_scratch/cess-v2test_baselineForPerf.ne1024pg2_ne1024pg2.F2010-SCREAMv1.20230901/run/`
Error message:
```
 7613: WARNING:CAAR: dp3d too small. k=30, dp3d(k)=43.483604, dp0=379.881287
 7613: WARNING:CAAR: dp3d too small. k=30, dp3d(k)=43.483604, dp0=379.881287
 7613: WARNING:CAAR: dp3d too small. k=30, dp3d(k)=43.483604, dp0=379.881287
 7613: WARNING:CAAR: dp3d too small. k=30, dp3d(k)=43.483604, dp0=379.881287
    0: bfbhash>           1476  2f98794acb30270 (Hommexx)
 7613:
 7613:  FAIL:
 7613: false
 7613: /autofs/nccs-svm1_home1/terai/SCREAM/code_w_perf_fix/components/eamxx/src/share/atm_process/atmosphere_process.cpp:442
 7613: Error! Failed post-condition property check (cannot be repaired).
 7613:   - Atmosphere process name: homme
 7613:   - Property check name: T_mid within interval [100, 500]
 7613:   - Atmosphere process MPI Rank: 7613
 7613:   - Message: Check failed.
 7613:   - check name: T_mid within interval [100, 500]
 7613:   - field id: T_mid[Physics PG2] <double:ncol,lev>(1536,128) [K]
 7613:   - minimum:
 7613:     - value: -1264.52
 7613:     - indices (w/ global column index): (24011477,126)
 7613:     - lat/lon: (56.2953, 236.449)
 7613:     - additional data (w/ local column index):
 7613:
...
```

PeterCaldwell commented:
Yikes, it sounds like Sarat's performance branch causes the model to go unstable after 1.5 days. Both CAAR dp3d too small and T>500K are not what we want to see. Perhaps we should retreat to just rocm5.1?

sarats commented:
I'm not aware of anything that should change numerics in my branch. It applies O2 for CXX explicitly which I presume is okay and is already default in EKAT and the ice files don't have opt on. Anyway, would be curious to see  the root cause of any numerical stability.
